pkg/cmd/cli/describe/describer.go

```diff
@@ -367,6 +367,7 @@ func describeCustomStrategy(s *buildapi.CustomBuildStrategy, out *tabwriter.Writ
 // DescribeTriggers generates information about the triggers associated with a buildconfig
 func (d *BuildConfigDescriber) DescribeTriggers(bc *buildapi.BuildConfig, out *tabwriter.Writer) {
 	describeBuildTriggers(bc.Spec.Triggers, out)
+	formatString(out, "\nBuilds Schedule", buildapi.SchedulePolicyType(bc.Spec.SchedulePolicy))
```

Comment:
@bparees it will but not if the SchedulePolicy is `nil`, so I need a function to set the default.